From 7e0b6f0a96477c5d12434231ea413d3a16658ed0 Mon Sep 17 00:00:00 2001
From: Abderrahim Kitouni <akitouni@gnome.org>
Date: Tue, 18 Dec 2018 10:54:06 +0100
Subject: [PATCH] rename g_strv_equal to gkbd_strv_equal

it conflicts with the newly added g_strv_equal in glib
---
 libgnomekbd/gkbd-keyboard-config.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libgnomekbd/gkbd-keyboard-config.c b/libgnomekbd/gkbd-keyboard-config.c
index bbe6904..00dc429 100644
--- a/libgnomekbd/gkbd-keyboard-config.c
+++ b/libgnomekbd/gkbd-keyboard-config.c
@@ -52,7 +52,7 @@ const gchar *GKBD_KEYBOARD_CONFIG_ACTIVE[] = {
  */
 
 static gboolean
-g_strv_equal (gchar ** l1, gchar ** l2)
+gkbd_strv_equal (gchar ** l1, gchar ** l2)
 {
 	if (l1 == l2)
 		return TRUE;
@@ -555,7 +555,7 @@ gkbd_keyboard_config_equals (GkbdKeyboardConfig * kbd_config1,
 	    (kbd_config2->model != NULL) &&
 	    g_ascii_strcasecmp (kbd_config1->model, kbd_config2->model))
 		return False;
-	if (!g_strv_equal (kbd_config1->layouts_variants,
+	if (!gkbd_strv_equal (kbd_config1->layouts_variants,
 			   kbd_config2->layouts_variants))
 		return False;
 
-- 
2.17.0

